pages/docs/integrations/grouparoo-intercom.mdx

```diff
@@ -11,132 +11,195 @@ Grouparoo's [Intercom](https://www.intercom.com/) Plugin enables you to export c
+  width={1625}
+  height={1305}
+/>
+<br></br>
+<br></br>
 
-```js
-options: {
-  creationMode: "Lifecycle",
-  removalMode: "Archive"
-},
-```
+#### Required Properties
 
-### Property Mappings
+Some properties are required to be able to find or create the correct Contacts in Intercom.
 
-These mappings are what tell Grouparoo which Properties you want to sync to the Destination and what they should be called. For this Plugin, **Grouparoo Properties** map to **Intercom People Attributes**.
+In Lead creation mode, the only required field for mapping Grouparoo Properties to Intercom User Attributes is **email**. Otherwise, when using User or Lifecycle creation mode, there are two required fields to map: **email** and** external_id**.
 
-For Intercom, Mapping the `email` Property is always required. If `creationMode` is set to `User` or `Lifecycle`, `email` is also a required Mapping.
+- **email: **a Contact’s email address in Intercom
+- **exernal_id**: set on a Contact with an identifier from your internal system, often assigned when they sign up for your service
 
-You can also map to custom attributes that you've defined on Intercom by adding the `custom_attributes` prefix (e.g. `custom_attributes.my_custom_attribute`).
+#### Optional Properties
 
-Here's an example of how this would look in the config file. Note that the keys refer to Intercom Attribute names and values refer to Grouparoo Property IDs.
+In addition to required fields, Intercom has many other User Attributes available to map data to including custom attributes that you have already created. Any of these can be synced to from Grouparoo Properties.
 
-```js
-mapping: {
-  email: "email",
-  name: "firstName",
-  "custom_attributes.lifetime_value": "ltv",
-},
-```
+### Group Data
 
-### Group Mappings
+In addition to mapping Grouparoo Properties to Intercom User Attributes, you can use Grouparoo Groups to apply tags to Intercom User Records.
 
-You can use **Group Memberships** to automatically add **Intercom Tags** to your contacts.
+In Intercom, tags are similar to User Attributes and segments but they are used for grouping users and companies manually to highlight special features such as those requiring VIP support or those who are loyal customers. Tags can be synced using Grouparoo Groups.
 
-Here's an example of how to configure this in the config file. Keys refer to the name of the tag to be shown on Intercom and values refer to the Grouparoo Group ID.
+<ImageInBrowserFrame
+  src="/images/docs/integrations/intercom/intercom-group-data.png"
+  alt="Intercom Destination Options"
+  width={1625}
+  height={367}
+/>
 
-```js
-destinationGroupMemberships: {
-  "High Value Customers": "highValueCustomers",
-},
-```
+## Notes
 
-## Intercom Next Steps
+We strive to keep our documentation accurate, easy to use, and up to date. If you notice that something has changed or is not addressed in the process of setting up your Grouparoo connection with Intercom, please let us know.
```

Comment:
Nice!  It would probably be helpful to make "let us know" a link... maybe suggest they email`help@grouparoo.com`?

Reply:
Perfect, that's been added.